docs/getting-started.md

```
@@ -91,6 +91,32 @@ yay -S spicetify-cli
 
 Depending on how you installed Spotify, you may need additional configuration.
 
+<details>
+<summary>Setting spicetify config to path</summary>
+
+Determine your shell
+   
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)